privacy.md

```diff
@@ -3,20 +3,286 @@ layout: AboutPage
+privacy and data protection. You may also, of course, file a complaint with the
+competent regulatory authorities.
+
+#### Analytics and third-party tools
+
+When visiting our website, statistical analyses may be made of your surfing behavior.
+This happens primarily using cookies and analytics. The analysis of your surfing
+behavior is usually anonymous, i.e. we will not be able to identify you from this data.
+You can object to this analysis or prevent it by not using certain tools. Detailed
+information can be found in the following privacy policy.
+
+You can object to this analysis. We will inform you below about how to exercise your
+options in this regard.
+
+## General information and mandatory information
+
+### Data protection
+
+The operators of this website take the protection of your personal data very seriously.
+We treat your personal data as confidential and in accordance with the statutory data
+protection regulations and this privacy policy.
+
+If you use this website, various pieces of personal data will be collected. Personal
+information is any data with which you could be personally identified. This privacy
+policy explains what information we collect and what we use it for. It also explains
+how and for what purpose this happens.
+
+Please note that data transmitted via the internet (e.g. via email communication) may be
+subject to security breaches. Complete protection of your data from third-party access
+is not possible.
+
+### Notice concerning the party responsible for this website
+
+The party responsible for processing data on this website is:
+
+<p>openHAB Foundation e.V.<br />Kollwitzweg 10<br />64372 Ober-Ramstadt</p>
+</address>
+<p><i class="material-icons">phone</i> +49 (6154) 603-9765</p>
+<p><i class="material-icons">email</i> <a href="mailto:info@openhabfoundation.org">info@openhabfoundation.org</a></p>
+
+The responsible party is the natural or legal person who alone or jointly with others
+decides on the purposes and means of processing personal data (names, email addresses,
+etc.).
+
+### Revocation of your consent to the processing of your data
+
+Many data processing operations are only possible with your express consent. You may
+revoke your consent at any time with future effect. An informal email making this
+request is sufficient. The data processed before we receive your request may still be
+legally processed.
+
+### Right to file complaints with regulatory authorities
+
+If there has been a breach of data protection legislation, the person affected may file
+a complaint with the competent regulatory authorities. The competent regulatory authority
+for matters related to data protection legislation is the data protection officer of the
+German state in which our company is headquartered. A list of data protection officers
+and their contact details can be found at the following link: https://www.bfdi.bund.de/DE/Infothek/Anschriften_Links/anschriften_links-node.html.
```

Comment:
These URLs are not transformed into hyperlinks, I'll fix them later.

Reply:
@ghys Sorry about the URLs. The problem with the URLs were caused by my Markdown previewer. It automatically converted them into links, so I forgot to format them correctly :-(.

- Google Analytics: I think you can use the same code. Please make sure that the anonymize_ip option is set to true.
- This is a good question. If we use this page also for the community you can simply add the text from section "Registration on this Website" from this document: https://docs.google.com/document/d/1No9_GC6urV7pkKR5KlSsovJI4bX6TS9XG6OoOKUOjLU/edit?usp=sharing